README.md

```diff
@@ -198,6 +198,10 @@ With reference to the [GitHub Commit status API], the `POST` parameters (`state`
   The log level (one of `debug`, `info`, `warn`, `error`, `silent`).\
   Default: `info`.
 
+- `github_api_endpoint`:\
```

Comment:
The term "endpoint" means a specific URL (each API has N endpoints, all on the same hostname), so here it would be misleading.

To find a more precise name, we could use the same terminology as the GH cli tool `gh`:

```
$ gh help auth login
Authenticate with a GitHub host.
...
FLAGS
  -h, --hostname string       The hostname of the GitHub instance to authenticate with
...
EXAMPLES
  # authenticate with a specific GitHub instance
  $ gh auth login --hostname enterprise.internal
```

So we could call this parameter `github_hostname`.

Reply:
Changed as suggested.